discord_connector.py

```diff
@@ -339,6 +339,46 @@ def parse_message_flags(self, message):
 
         return true_flags
 
+    def _handle_delete_message(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        channel_id = param['channel_id']
+        message_id = param['message_id']
+
+        message, ret_val = self._async_loop.run_until_complete(self.delete_message(channel_id, message_id))
+        self.save_progress("Action on message {} ended with return value of: {}: {}".format(message_id, ret_val, message))
+
+        summary = action_result.update_summary({})
+        summary['action result: '] = "Action on message {} ended with return value of: {}: {}".format(message_id, ret_val, message)
+
+        if ret_val != 200:
+            return action_result.set_status(phantom.APP_ERROR)
+        return action_result.set_status(phantom.APP_SUCCESS)
```

Comment:
so basically this could be:

```python
return action_result.set_status(ret_val == 200)
```

FYI: yes, APP_SUCCESS is simply `True` and APP_ERROR is `False`